https://github.com/zlin/wgetpaste/pull/45

From ab4632ca9cd54c9e7478f37eee821113c0135c37 Mon Sep 17 00:00:00 2001
From: Oskari Pirhonen <xxc3ncoredxx@gmail.com>
Date: Fri, 28 Jul 2023 18:55:10 -0500
Subject: [PATCH] Disable sprunge

Sprunge seems to have recently died. Disable it by removing it from the
service list, but keep the core bits in case it decides to come back.

RIP ;(
---
 wgetpaste | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wgetpaste b/wgetpaste
index 864d5ef..b3a537b 100755
--- a/wgetpaste
+++ b/wgetpaste
@@ -41,7 +41,7 @@ POST_generic() {
 }
 
 ### services
-SERVICES="0x0 bpaste codepad dpaste gists ix_io snippets sprunge"
+SERVICES="0x0 bpaste codepad dpaste gists ix_io snippets"
 # 0x0
 ENGINE_0x0=0x0
 URL_0x0="http://0x0.st"
-- 
2.41.0

